.gitmodules

```diff
@@ -0,0 +1,6 @@
+[submodule "assets/Font-Awesome"]
+	path = assets/Font-Awesome
+	url = https://github.com/FortAwesome/Font-Awesome.git
+[submodule "assets/bootstrap-sass"]
+	path = assets/bootstrap-sass
+	url = https://github.com/twbs/bootstrap-sass
```

Comment:
Instead of using submodules you can use the https://github.com/jhthorsen/mojolicious-plugin-assetpack/blob/master/lib/Mojolicious/Plugin/AssetPack/Guides/Cookbook.pod#bootstrap

But that's up to you. I just don't like submodules, hehe